### ISSUE TITLE
bookmarks in .settings note card

### ISSUE THREAD

**Issue by Trotmistwalker** [OpenCollarTeam/OpenCollar#577]
What version of OpenCollar are you using? 8.0.5

What behavior did you expect? when the remove bookmark is pressed by the owner (Dominant) all bookmarks should be displayed not just the ones on the .bookmarks note card

What behavior did you see instead? when remove bookmark is pressed by the Dominant only the bookmarks on the .bookmarks note card are displayed as options to remove

What steps does someone need to take to reproduce the problem? have bookmarks on both the .settings note card and the .bookmarks note card
When the bookmarks are saved on the .bookmarks note card and some in the .settings then the bookmarks are all visible to the Dominant but when they wish to remove a bookmark only the ones on the .bookmarks come up to be removed. not the ones in the .settings note card.

**Comment by Medea-Destiny** [comment]
The .bookmarks card is for setting "permanent" bookmarks. The volatile list (stored in script settings and may be dumped to the settings notecard, but shouldn't be) are temporary ones that can be added and removed via the menus. 

I don't have a .bookmarks notecard, and the remove option gives me all the volatile landmarks to remove exactly as expected, so I can't reproduce this issue. As scripts cannot edit notecards, there's no way for the collar to permanently removing bookmarks from the .bookmarks notecard, which gets reloaded each time inventory changes for example. Such removals have to be done by editing the notecard. This is all working as intended as far as I can see.

Really you shouldn't have bookmarks stored in your .settings notecard. If they are stored there, then when you delete them from bookmarks via the menu, they will be restored next time the .settings notecard is loaded, which is not intended behaviour. Perhaps we should consider some informational text when doing a settings dump to clarify this point. 

To summarize; the Remove menu will not show bookmarks that are stored in the .bookmarks notecard, unless you have some overlap where there are bookmarks that are stored in .bookmarks and additionally in your .settings card, which may be a cause of confusion. It should only show those bookmarks that have been added manually. Can you confirm this is indeed what you're seeing please?

**Comment by SilkieSabra** [comment]
This actually raises a good question for me, because a few versions back I noticed that bookmarks could be stored in settings and there didn't seem to be a purpose for a separate .bookmarks notecard, which has not gone standard into the collar for quite some time.  If there is a separate notecard then bookmarks should not be stored in both places.

**Comment by Medea-Destiny** [comment]
Warning text added to oc_settings in https://github.com/OpenCollarTeam/OpenCollar/tree/Medea-4.1-1

**Comment by AndurFaulds** [comment]
I am noticing corruption issues relating to this but not exactly this issue. I Save a Bookmark and use Uppercase and lower case 'Bookmark Name'. after closing and reopening Bookmarks it changes the case to all lower case 'bookmark name" Erratically it also duplicates said marks. I create Bookmarks named 'A Mark' 'B Mark' 'C Mark' then I open it and see those 3 PLUS 'a mark' 'b mark' 'c mark' OR other times it erases the first three marks and keeps the last three marks, sometimes two marks.

**Comment by SilkieSabra** [comment]
has this been addressed and is it even an issue or an artifact of people storing bookmarks in the settings notecard?
